src/components/ButtonPrimary.js

```diff
@@ -12,8 +17,13 @@ type Props = {
 };
 
 const Button = ({ children, disabled, onPress }: Props) => (
-  <Touchable style={styles.button} onPress={onPress} disabled={disabled}>
-    <Text type="h2" style={styles.text}>
+  <Touchable
+    style={[styles.button, disabled ? styles.disabledButton : {}]}
+    onPress={onPress}
+    disabled={disabled}
+    accessibilityTraits={disabled ? ["button", "disabled"] : ["button"]}
```

Comment:
It could be nice to move this logic into `Touchable.ios.js`. Adding the `disabled` could be generally useful.